### ISSUE TITLE
Using istioctl upgrade to 1.5.0, pilots permission denied

### ISSUE THREAD

**Issue by erdun** [istio/istio#22312]
Using istioctl upgrade from 1.4.6 to 1.5.0

Ths ServiceAccount in clusterRoleBinding istio-pilot-istio-system is replaced from `istio-pilot-service-account` to  `istiod-service-account`, leads to the origin pilots permission denied

**Comment by erdun** [comment]
Clusterrole istio-pilot-{{ .Release.Namespace }} in 1.4.6 
```
- apiGroups: ["apiextensions.k8s.io"]
  resources: ["customresourcedefinitions"]
  verbs: ["*"]
```

in 1.5.0 is 
```
- apiGroups:
  - apiextensions.k8s.io
  resources:
  - customresourcedefinitions
  verbs:
  - get
  - watch
  - list
```

cannot create resource "customresourcedefinitions"

**Comment by howardjohn** [comment]
Why do you need to create `customresourcedefinitions`?

**Comment by howardjohn** [comment]
I think this was fixed in 1.5.2, we are now using istio-pilot-service-account again
